src/detail/detail.js

```diff
@@ -0,0 +1,69 @@
+import { LitElement, html } from "lit-element";
+import styles from "./detail.style.js";
+
+
+class DetailComp extends LitElement {
+
+  static get styles() {
+    return [styles];
+  }
+
+  static get properties() {
+    return {
+      cover: { book_title: Object }
+    }
+  }
+
+  constructor() {
+    super();
+    this.cover = {
+      sprites: {},
+      author: "",
+      book_title: []
```

Comment:
Why the book_title is received in an array?

Reply:
Done!